### ISSUE TITLE
Incompatible with `expect` 27.5.1

### ISSUE THREAD

**Issue by GerkinDev** [just-jeb/jest-marbles#401]
Hi,

In expect v`27.5.1` (thus jest with same version), `exports` are mentioned in `package.json`, and `jasmineUtils.js` is not in it.

Here are the sample `package.json`:

```json
{
  "name": "expect",
  "version": "27.5.1",
  "repository": {
    "type": "git",
    "url": "https://github.com/facebook/jest.git",
    "directory": "packages/expect"
  },
  "license": "MIT",
  "main": "./build/index.js",
  "types": "./build/index.d.ts",
  "exports": {
    ".": {
      "types": "./build/index.d.ts",
      "default": "./build/index.js"
    },
    "./package.json": "./package.json",
    "./build/utils": "./build/utils.js",
    "./build/matchers": "./build/matchers.js"
  },
}
```

Moreover, this file is removed from `^28.0.0` branch (currently in alpha).

Cheers !

**Comment by GerkinDev** [comment]
@PhilippMeissner good catch for the fix in the other package. I guess making a patch applicable via `patch-package` is pretty easy from there, waiting for the owner to ship a new release

**Comment by just-jeb** [comment]
Can't believe I missed this issue 🤦🏿‍♂️.
Will provide the fix soon, stay tuned.

**Comment by just-jeb** [comment]
Fixed in `3.0.3`, please confirm.
